src/types/FateAddresses.js

```diff
@@ -0,0 +1,27 @@
+const { FateAddress, FateAddressRaw } = require('./FateAddress.js')
+
+module.exports = [
+    ['Account', 'account_address', 'ak'],
+    ['Channel', 'channel_address', 'ch'],
+    ['Contract', 'contract_address', 'ct'],
+    ['Oracle', 'oracle_address', 'ok'],
+    ['OracleQuery', 'oracle_query_address', 'oq'],
+].reduce((prev, [name, ...args]) => {
```

Comment:
Combining these types into a single file to simplify the making of changes, not sure is it acceptable. But I don't think these types got a custom behavior in a foreseen future.